src/node/desktop/src/main/detect-r.ts

```diff
@@ -202,7 +203,7 @@ export function detectREnvironment(rPath?: string): Expected<REnvironment> {
   }
 
   // generate small script for querying information about R
-  const rQueryScript = String.raw`writeLines(c(
+  const rQueryScript = String.raw`writeLines(sep = "\x1F", c(
```

Comment:
Maybe paranoid, but using a custom delimiter just to avoid collisions with whatever output R happens to produce below. (Including newlines, if an error occurs, and R prints a bunch of "stuff"?)